### PR TITLE
Create the New Learn Structure for Swan Lake Beta3

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Create the new learn structure for Swan Lake Beta3.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: _data/clidocumentationnavswanlake.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - title: Update Tools
+      url: /learn/cli-documentation/update-tool/
+      active: update-tool
+    - title: Open API
+      url: /learn/cli-documentation/openapi/
+      active: openapi
+    - title: GRPC
+      url: /learn/cli-documentation/grpc/
+      active: grpc
+
```
Comment (anupama-pathirage) [inline _data/clidocumentationnavswanlake.yml:16]: Remove the new liens

File: _data/guidesnavswanlake.yml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      active: managing-dependencies
+
+- title: Publishing packages to Ballerina Central
+  url: /learn/publishing-packages-to-ballerina-central/
+  active: publishing-packages-to-ballerina-central
+
+- title: Calling Java Code from Ballerina and Vice Versa
+  url: /learn/calling-java-code-from-ballerina-and-vice-versa/
+  active: calling-java-code-from-ballerina-and-vice-versa
+
```
Comment (anupama-pathirage) [inline _data/guidesnavswanlake.yml:113]: Remove extra new lines, check all files

File: _data/clidocumentationnavswanlake.yml (hunk 1 above)
Comment (praneesha) [reply _data/clidocumentationnavswanlake.yml:16]: Done

File: _data/guidesnavswanlake.yml (hunk 2 above)
Comment (praneesha) [reply _data/guidesnavswanlake.yml:113]: Done

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**